README.md

```diff
@@ -1,3 +1,13 @@
 # Chaos-Charts
 This repository hosts the Litmus Chaos Charts.
 
+## Installation Steps for Chart Releases
```

Comment:
Can we also mention a comment stating that it will applicable only for `1.1.0 release` or  onward?

Reply:
done